install_config/topics/aws_service_account.adoc

```diff
@@ -0,0 +1,117 @@
+                "elasticloadbalancing:CreateLoadBalancerListeners",
+                "elasticloadbalancing:ConfigureHealthCheck",
+                "elasticloadbalancing:DeleteLoadBalancerListeners",
+                "elasticloadbalancing:RegisterInstancesWithLoadBalancer",
+                "elasticloadbalancing:DescribeLoadBalancers",
+                "elasticloadbalancing:CreateLoadBalancer",
+                "elasticloadbalancing:DeleteLoadBalancer",
+                "elasticloadbalancing:ModifyLoadBalancerAttributes",
+                "elasticloadbalancing:DescribeLoadBalancerAttributes"
+            ],
+            "Resource": "*",
+            "Effect": "Allow",
+            "Sid": "1"
+        }
+    ]
+}
+----
+
+
+[source,yaml]
+----
+aws iam put-role-policy \
+  --role-name openshift-role \
+  --policy-name openshift-admin \
+  --policy-document file: //openshift_iam_policy
+----
+
+[source,yaml]
+----
+aws iam put-user-policy \
+  --user-name openshift-admin \
+  --policy-name openshift-admin \
+  --policy-document file: //openshift_iam_policy
+----
+
+NOTE: The OpenShift node instances only need the `ec2:DescribeInstance` 
+permission but the installer only allows for a single AWS access key and secret 
+to be defined. This can be bypassed using IAM roles and assigning the permissions 
+above to the master instances and the `ec2:DescribeInstance` to nodes.
+
+=== Configuring the {product-title} cloud provider at installation
+
+.Procedure
+
+To configure the configure the Amazon Web Services cloud provider using an IAM account
+with an access and secret key add the following values to the inventory:
+
+[source,yaml]
+----
+[OSEv3:vars]
+openshift_cloudprovider_kind=aws
+openshift_clusterid=openshift <1>
+openshift_cloudprovider_aws_access_key=AKIAJ6VLBLISADPBUA <2>
+openshift_cloudprovider_aws_secret_key=g/8PmDNYHVSQn0BQE+xtsHzbaZaGYjGNzhbdgwjH <3>
+----
+<1> A tag assigned to all resources (instances, load balancers, vpc, etc) used for OpenShift.
+<2> AWS access key used by the IAM account.
+<3> AWS secret key used by the IAM account.
+
+To configure the configure the Amazon Web Services cloud provider using an IAM role
+ add the following values to the inventory:
+
+ [source,yaml]
+ ----
+ [OSEv3:vars]
+ openshift_cloudprovider_kind=aws
+ openshift_clusterid=openshift <1>
+ ----
+ <1> A tag assigned to all resources (instances, load balancers, vpc, etc) used for OpenShift.
+
+ NOTE: The IAM role takes the place of needing an access and secret key.
+
+=== Configuring the {product-title} cloud provider after installation.
+
+In the event that the Amazon Web Services cloud provider values were not provided at installation time
+the configuration can be defined and created after the installation. Follow the steps to configure the
+configuration file and manually configuring the master and node.
+
+xref:../install_config/install_config/configuring_aws.adoc#aws-configuring-masters-manually
```

Comment:
This path is just hanging out by itself. Is it supposed to be in the paragraph above?